tests/test_sd_whonix.py

```diff
@@ -54,9 +54,6 @@ def test_sd_whonix_repo_enabled(self):
         """
         assert self._fileExists(self.whonix_apt_list)
 
-    def test_logging_configured(self):
-        self.logging_configured(vmname=True)
-
```

Comment:
The `test_logging_configured` shouldn't be removed, it should just have its call updated to pass no args (matching the modified test logic).

Reply:
Fixed (I misread the inheritance logic).